instance/models/openedx_appserver.py

```diff
@@ -278,9 +278,12 @@ def make_active(self, active=True):
         self.is_active = active
         self.save()
         self.instance.reconfigure_load_balancer()
+        appserver_num = self.name[-1]
```

Comment:
@pkulkark, is this logic correct? For example, an instance could have an active appserver with name 'AppServer 17` and this will return `7`.

Reply:
Oops! sorry looks like I pushed the wrong change.